perfectextractor/apps/extractor/base.py

```diff
@@ -112,18 +143,8 @@ def collect_file_names(self, dir_name):
         return file_names
 
     def generate_results(self, dir_name):
-        results = []
-
-        for file_name in self.collect_file_names(dir_name):
-            result = self.process_file(file_name)
-
-            for extractor in self.other_extractors:
```

Comment:
@mhkuu I think this didn't work as expected previously. Is it an important feature?

Reply:
No, this can go for the moment!